packages/cel/src/list.ts

```diff
@@ -172,3 +172,5 @@ function celFromListElem(desc: DescField & { fieldKind: "list" }, v: unknown) {
       return celFromScalar(desc.scalar, v as ScalarValue);
   }
 }
+
+export const EMPTY_LIST = celList([]);
```

Comment:
Adding this to the top, it complained about using the Implementation class before it was declared.